charts/fission-all/values.yaml

```diff
@@ -361,6 +361,34 @@ controller:
     runAsUser: 10001
     runAsGroup: 10001
 
+## canaryconfig is the component that the client talks to.
+## It contains canaryconfig feature and there handlings
+##
+canaryconfig:
+  ## Pod resources as:
```

Comment:
Can we move all of this section under existing `canaryDeployment` ?

So that we don't have two sections for one component.